### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kabilar**
- Add collapsible sections
- Add citation section
- Move background file to elements.datajoint.org
- Add links to elements.datajoint.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +23,10 @@ See [Background](Background.md) for the background information and development t
 
      + `Location` as a dependency for `ScanLocation`
 
+## Table definitions
+<details>
```
Comment (CBroz1) [inline README.md:27]: I would limit click-to-expand format to cases where a user may not need this information. When it encompasses a whole section, it doesn't do much to facilitate navigation. In general, I would adopt the following structure

```
- Item 1. General item description. <details>
<summary>Click to expand details</summary>
   - Detail 1
   - Detail 2
</details>
- Item 2 General item description. <details>
<summary>Click to expand details</summary>
   - Detail 1
   - Detail 2
</details>
```  

For the current doc, I think the items could be (a) schema with collapsed table definitions and (b) installation methods, such as 'Install with Suite2p'

**Review by CBroz1 — APPROVED** [review]
Suggesting a big fix regarding `<details>` structure. If we don't want to tackle this in the current PR, we can open an issue on the topic.

File: README.md (hunk 1 above)
Comment (kabilar) [reply README.md:27]: Thanks @CBroz1.  I agree.  Will update this pull request.

**Comment by kabilar** [comment]
Hi @CBroz1, I have incorporated your suggestions.  Thanks.

File: README.md [hunk 2]
```diff
@@ -114,6 +112,8 @@ The `imaging` schema stores information regarding the calcium traces for each ma
 
      + If your workflow uses these packages, you should install them when you install `element-interface`.
 
+     <details>
+     <summary>Click to expand details</summary>
```
Comment (CBroz1) [inline README.md:116]: I looks like backtic formatting isn't working properly in some cases. Adding an extra line between `</summary>` and `+ Bulleted item` seems to help. 

Current screenshot:
> Screen Shot 2022-04-26 at 09 09 57

With extra line:
> Screen Shot 2022-04-26 at 09 23 44

**Review by CBroz1 — CHANGES_REQUESTED** [review]
Minor edit: add extra line after each `</summary>` tag

**Comment by CBroz1** [comment]
Minor additional comment - the updated diagram is large and would benefit from [ratio adjustment](https://github.com/datajoint/datajoint-python/issues/975). If not this PR, I'll open an issue and self-assign

**Comment by kabilar** [comment]
> Minor additional comment - the updated diagram is large and would benefit from [ratio adjustment](https://github.com/datajoint/datajoint-python/issues/975). If not this PR, I'll open an issue and self-assign

Good idea.  I created an issue (https://github.com/datajoint/element-calcium-imaging/issues/74).  Let's address in a future pull request.  Thanks.

File: README.md (hunk 2 above)
Comment (kabilar) [reply README.md:119]: Good catch!  I have fixed the four locations with this issue.

**Comment by kabilar** [comment]
Thanks for your review, @CBroz1.  I have updated this pull request.

**Pull request merged**